### PR TITLE
Lucah endicott  host update session states

### PR DESCRIPTION
**Pull request by lucahendicott**

### CODE REVIEW

File: debug-panel/src/App.tsx [hunk 1]
```diff
@@ -20,6 +20,24 @@ function App() {
...
+    if (gameSession == null) {
+            return
+          }
+          
+    let gameSessionId = gameSession!.id
+    
+    apiClient.updateGameSession(gameSessionId, gameSessionState)
+    .then(gameSession => {
+    setUpdatedGameSession(gameSession)
+    setError(null)
```
Comment (maniramezan) [inline debug-panel/src/App.tsx:34]: nit: indentation

File: debug-panel/src/App.tsx [hunk 2]
```diff
@@ -20,6 +20,24 @@ function App() {
...
+    }).catch(error => {
+    console.error(error.message)
+    setError(error.message)
+    setUpdatedGameSession(null)
```
Comment (maniramezan) [inline debug-panel/src/App.tsx:38]: Same as above

File: debug-panel/src/App.tsx [hunk 3]
```diff
@@ -20,6 +20,24 @@ function App() {
     return () => subscription.unsubscribe()
 }, []);
 
+const handleUpdateGameSessionState = (gameSessionState: GameSessionState) => 
+  {
+    if (gameSession == null) {
+            return
+          }
```
Comment (maniramezan) [inline debug-panel/src/App.tsx:27]: Same as below nit

File: debug-panel/src/App.tsx (hunk 1 above)
Comment (lucahendicott) [reply debug-panel/src/App.tsx:34]: 100. Will fix

**Pull request merged**